### DISCUSSION TITLE
Why did I execute these two commands over time? No firewall, port 9878 also listens:

### DISCUSSION THREAD

**Discussion by a28124791** [apache/ozone#7231] [FAQ] [answered]
I started ozone on 192.168.95.111 using the following command
docker  run -tid -p 9878:9878 apache/ozone
The following environment variables are configured on the s3 client machine:
AWS_ACCESS_KEY_ID=testuser/scm@EXAMPLE.COM
AWS_SECRET_ACCESS_KEY=xyz
Why did I execute these two commands over time? No firewall, port 9878 also listens:
aws s3api --endpoint http://192.168.95.111:9878 create-bucket --bucket buckettest
aws s3 ls --endpoint http://192.168.95.111:9878 s3://buckettest
C:\Users\Administrator>aws s3api --endpoint http://192.168.95.111:9878 create-bucket --bucket buckettest

Read timeout on endpoint URL: "http://192.168.95.111:9878/buckettest"

C:\Users\Administrator>aws s3 ls --endpoint http://192.168.95.111:9878 s3://buckettest

Both ping and telnet are accessible.

**Comment by adoroszlai** [comment, marked as answer]
Please try `http://localhost:9878` as endpoint in your command.
